deployment-configuration.md

```diff
@@ -37,7 +39,20 @@ and Unicorn on Ubuntu 16.
 
 2. Setup the crucible_smart_app.
 
-Issue the following commands to setup the crucible smart app.
+To use all features, install Ruby 2.5. Enter the following commands in your terminal.
```

Comment:
Maybe say something like 

TLS connection testing requires Ruby 2.5 or greater.  To check to see what version of ruby is installed, type in the following command:

```
ruby --version
```
If you are not running version Ruby 2.5, you can install it using Ruby Version Manager (rvm) by issuing the following commands.